doc/libpmem2/pmem2_config_set_address.3.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: manual
+Content-Style: 'text/css'
+title: _MP(PMEM2_CONFIG_SET_ADDRESS, 3)
+collection: libpmem2
+header: PMDK
+date: pmem2 API version 1.0
+...
+
+[comment]: <> (SPDX-License-Identifier: BSD-3-Clause)
+[comment]: <> (Copyright 2020, Intel Corporation)
+
+[comment]: <> (pmem2_config_set_address.3 -- man page for libpmem2 config API)
+
+[NAME](#name)<br />
+[SYNOPSIS](#synopsis)<br />
+[DESCRIPTION](#description)<br />
+[RETURN VALUE](#return-value)<br />
+[SEE ALSO](#see-also)<br />
+
+# NAME #
+
+**pmem2_config_set_address**() - set address among with address request type in the
+pmem2_config structure
+
+# SYNOPSIS #
+
+```c
+#include <libpmem2.h>
+
+struct pmem2_config;
+int pmem2_config_set_address(struct pmem2_config *cfg, void *addr, int request_type);
+```
+
+# DESCRIPTION #
+
+The **pmem2_config_set_address**() function configures the address among with address request
```

Comment:
... sets the starting address *addr* which will be used as a hint for memory mapping. If the *addr* is not specified in the config, the starting address will be chosen by the operating system. The *type* specifies how strictly the hint address should be enforced.